addon/-private/execution_context/acceptance.js

```diff
@@ -60,6 +61,30 @@ AcceptanceExecutionContext.prototype = {
     triggerEvent(selector, container, eventName, eventOptions);
   },
 
+  focus(selector, options) {
+    let element = this.findWithAssert(selector, options)
```

Comment:
Suggest the following, for consistency with existing code (see [here](https://github.com/san650/ember-cli-page-object/pull/351/files#diff-49dc265ae7bed20f928377a296d9c3a7R94) and [here](https://github.com/san650/ember-cli-page-object/blob/master/addon/-private/execution_context/native-events-context.js#L73), for example):
```
let element = this.findWithAssert(selector, options)[0];
```
or
```
let $selection = this.findWithAssert(selector, options);
```